### PR TITLE
[script.module.tzdata] 2021.5+matrix.1

### PR DESCRIPTION
**Pull request by ronie**
### Description
required by https://github.com/xbmc/repo-scripts/pull/2172
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: script.module.tzdata/addon.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+<addon id="script.module.tzdata" name="tzdata" version="2021.5+matrix.1" provider-name="pganssle">
+  <requires>
+    <import addon="xbmc.python" version="3.0.0"/>
+  </requires>
+  <extension point="xbmc.python.module" library="lib"/>
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en_GB">zic-compiled binaries for the IANA time zone database</summary>
+    <description lang="en_GB">tzdata is intended to be a fallback for systems that do not have system time zone data installed.</description>
+    <platform>all</platform>
+    <language></language>
```
Comment (enen92) [inline script.module.tzdata/addon.xml:11]: can you drop the language tag?

File: script.module.tzdata/addon.xml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    <license>Apache-2.0</license>
+    <forum></forum>
```
Comment (enen92) [inline script.module.tzdata/addon.xml:13]: can you drop the forum tag?

File: script.module.tzdata/addon.xml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.module.tzdata" name="tzdata" version="2021.5+matrix.1" provider-name="pganssle">
```
Comment (enen92) [inline script.module.tzdata/addon.xml:2]: if this won't be available below matrix, no need for the local version identifier

**Review by enen92 — CHANGES_REQUESTED** [review]
Minors

**Comment by ronie** [comment]
updated as requested

**Pull request merged**